### DISCUSSION TITLE
[Bug] General Question about Budgie Brightness Applet

### DISCUSSION THREAD

**Discussion by brent111** [BuddiesOfBudgie/budgie-desktop#618] [Q&A] [answered]
### Description

Actually a general question and not a bug that I know, please forgive me.

On my laptop Budgie (up to date) I have 3 adjustment tracks on the Brightness Controller Applet.
On my desktop Budgie (up to date) I have 2 adjustment tracks on the Brightness Controller Applet.
Both are Solus.

Why? Is there two versions or something? The laptop has much greater pinpoint brightness control while I struggle with the 2 tracks on the desktop.

How do I get the three-track package on the desktop budgie? Thank you.

### Budgie version

10.9.2

### Operating System

Solus 4.5

### Steps to reproduce the issue

1) Open Brightness Controller on Budgie Desktop.
2) See a 2-track brightness controller
1A) open brightness controller on budgie laptop
1B) see a three-track brightness controller

### Actual result

same budgies, two different brightness controllers

### Expected result

same budgies, same brightness applet controllers

**Comment by samlane-ma** [comment, marked as answer]
If you are referring to the Budgie Brightness Controller Applet, it is provided by Ubuntu Budgie:
https://github.com/UbuntuBudgie/budgie-extras/issues

However...

The applet uses 3 sliders. Going off the top of my head here...
1) Backlight control
2) Gamma control using xrandr
3) Blue level using xrandr

I am guessing you are missing # 1 on your desktop, which uses what is in "/sys/class/backlight" to adjust the backlight level.
With an external monitor, you wouldn't have this type of backlight control the way you do on a laptop (at least not as far as I know).

Basically, this is expected behavior.

**Comment by brent111** [comment]
thanks for the detailed explanation, I really appreciate the knowledge. Brightness control is much better on laptop, perhaps you will 'port' this to the desktop version someday down the road. I will close this. Thanks again.
